source/qdk_package/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools>=64", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "qdk"
+version = "0.0.0"
+description = "Unified Quantum Development Kit Python meta-package (wraps qsharp and optional extras)"
+readme = "README.md"
+authors = [ { name = "Microsoft" } ]
+license = { file = "LICENSE.txt" }
+requires-python = ">=3.9"
+dependencies = [
+  "qsharp==1.20.0",  # rely on existing package
+]
+
+[project.optional-dependencies]
+widgets = ["qsharp-widgets==1.20.0"]
+azure = ["azure-quantum==3.2.0"]
+qiskit = ["qiskit>=1.2.2,<2.0.0"]
```

Comment:
As it came up in our discussion earlier, I believe when the user specifies both optional features `qdk[azure, qiskit]`, we should also enable the `azure-quantum[qiskit]` optional feature (I think this installs more than just `qiskit`, based on what I'm seeing in [this file](https://github.com/microsoft/azure-quantum-python/blob/main/azure-quantum/requirements-qiskit.txt))  I have no clue how to express this in a `pyproject.toml` though.